client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -73,377 +73,466 @@ module.exports = protectForm( React.createClass( {
 	},
 
 	resetState: function() {
-		var clearSettings = { fetchingSettings: true };
+		const clearSettings = { fetchingSettings: true };
 		this.discussionAttributes.forEach( function( attribute ) {
 			clearSettings[ attribute ] = '';
 		} );
 		this.replaceState( clearSettings );
 		debug( 'resetting state' );
 	},
 
-	defaultArticleSettings: function() {
+	setDirtyField( key ) {
```

Comment:
I'm not familiar with 'dirty' terminology. If we are dropping `dirtyLinkedState`, can we rename this?